### ISSUE TITLE
link to Note Block

### ISSUE THREAD

**Issue by meichthys** [TriliumNext/trilium#4997]
zadam/trilium#1021

**Comment by FliegendeWurst** [comment]
We now have bookmarks, but I don't think you can link to them yet. https://github.com/orgs/TriliumNext/discussions/1972

**Comment by meichthys** [comment]
Yes, I was playing with this the other day. It looks like we can link two bookmarks if you are in the same note however linking to a bookmark in a different note does not currently work.
